api/stream.go

```diff
@@ -301,6 +302,32 @@ func (r streamRoutes) getStream(c *gin.Context) {
 		"vod":         stream.Recording})
 }
 
+func (r streamRoutes) getStreamPlaylist(c *gin.Context) {
+	type StreamPlaylistEntry struct {
+		StreamID   uint      `json:"streamId"`
+		CourseSlug string    `json:"courseSlug"`
+		StreamName string    `json:"streamName"`
+		LiveNow    bool      `json:"liveNow"`
+		Watched    bool      `json:"watched"`
+		CreatedAt  time.Time `json:"createdAt"`
+	}
+
+	tumLiveContext := c.MustGet("TUMLiveContext").(tools.TUMLiveContext)
+	var result []StreamPlaylistEntry
+	for _, stream := range tumLiveContext.Course.Streams {
```

Comment:
I think it would make sense to only include VODs in the playlist. Currently livestreams and streams in waiting rooms are displayed too.